translations/ja/LC_MESSAGES/wesnoth-ala.po

```diff
@@ -879,10 +865,8 @@ msgstr "川を渡る道をもう一度広げるんだ、そうして全員攻撃
 
 #. [unit]: type=Naga Myrmidon, id=Hisskss
 #: A_Little_Adventure/scenarios/06_The_Last_Battle.cfg:436
-#, fuzzy
-#| msgid "Hissss!"
 msgid "Hisskss"
-msgstr "シャーーーッ！"
+msgstr "シャーーーッ"
```

Comment:
So, this one is the name of a Naga...

Reply:
Ow!

Reply:
ok this one looks fixed now, although it appears you reverted your other changes in the process?

Reply:
Actually it's okay, I can just merge this one, and then you can submit another for the rest... (it needs to come out of draft mode first, though)

Reply:
God, forgive me!
I didn't know what to do, and I didn't know what was wrong, I changed to **draft** status.
I have just operated it and it is probably back to its original state, "**ready for review**" .

Reply:
it's ok! merging now